docs/docs/overview/quickstart/get-started-with-java.md

```diff
@@ -0,0 +1,336 @@
+Novu novu = new Novu(apiKey);
+
+TopicRequest createTopicRequest = new TopicRequest();
+    createTopicRequest.setKey("key");
+    createTopicRequest.setName("name");
+        try {
+            return novu.createTopic(createTopicRequest);
+        }catch (Exception e){
+            log.error("Error creating Topic", e);
+        }
+
+        return null;
+```
+
+Before running the code, make sure you understand the following:
+
+- When creating a `key`, ensure it is unique and accurately identifies the topic. Document naming conventions and communicate them to team members to avoid confusion and ensure a smooth workflow.
+- The value of `name` should be a descriptive topic name.
+
+### Add subscribers to a Topic
+
+Copy and paste the following code into your app to add subscribers a topic:
+
+```java
+String apiKey = "API_KEY";
+Novu novu = new Novu(apiKey);
+
+    SubscriberRequest subscriberRequest = new SubscriberRequest();
+        subscriberRequest.setEmail("email");
+        subscriberRequest.setFirstName("fName");
+        subscriberRequest.setLastName("lName");
+        subscriberRequest.setPhone("phone");
+        subscriberRequest.setAvatar("avatar");
+        subscriberRequest.setSubscriberId("subscriberId");
+
+        String topicKey = "key";
+        SubscriberAdditionRequest requestBody = new SubscriberAdditionRequest();
+        requestBody.setSubscribers(Collections.singletonList(subscriberRequest));
+
+        try {
+            return novu.addSubscriberToTopic(requestBody,topicKey);
+        }catch (Exception e){
+            log.error("Error Adding Subscriber To Topic", e);
+        }
+
+        return null;
+```
+
+On the other hand, if you want to remove subscribers from a topic, do the following:
+
+```java
+String apiKey = "API_KEY";
+Novu novu = new Novu(apiKey);
+
+    SubscriberRequest subscriberRequest = new SubscriberRequest();
+        subscriberRequest.setEmail("email");
+        subscriberRequest.setFirstName("fName");
+        subscriberRequest.setLastName("lName");
+        subscriberRequest.setPhone("phone");
+        subscriberRequest.setAvatar("avatar");
+        subscriberRequest.setSubscriberId("subscriberId");
+
+
+        String topicKey = "key";
+        SubscriberAdditionRequest requestBody = new SubscriberAdditionRequest();
+        requestBody.setSubscribers(Collections.singletonList(subscriberRequest));
+
+        try {
+            return novu.removeSubscriberFromTopic(requestBody, topicKey);
+        }catch (Exception e){
+            log.error("Error Removing Subscriber From Topic", e);
+        }
+
+        return null;
+```
+
+### Sending a Notification to a Topic
+
+Thanks to the topics feature, it is possible to trigger a notification to all subscribers assigned to a topic. This helps avoid listing all subscriber identifiers in the `to` field of the notification trigger.
+
+:::info
+This feature is not yet available on the Java SDK, this doc will be updated once it is released on Java.
+:::
```

Comment:
This feature is actually available on Java. Remember the `to` field can also accept a Topic or list of Topics

Reply:
Noted

Reply:
Resolved